### ISSUE TITLE
wild_MNIST/cfg.yaml hyperparameters

### ISSUE THREAD

**Issue by qichenglao** [autonomousvision/counterfactual_generative_networks#3]
Hi, thank you for the nice work! I was trying to train a CFG for the wild MNIST dataset using the provided hyperparameters (cgn_wildlife_MNIST/cfg.yaml). However, the accuracy (~76%, with much more epochs) is not comparable to that of your provided model (cgn_wildlife_MNIST/weights/ckp.pth) when training the invariant classifier. I am wondering if the cfg.yaml should be further fine-tuned, or is there anything that I should pay attention to? Thank you very much!
